Environment Page Redesign

vivianliu0 opened the pull request:
Demo: https://www.loom.com/share/562c2806a7e1494bb6b0476543512854

Demo of new changes after addressing the comments:
https://www.loom.com/share/89eec13bb03e4544a12b4dff38c70ec5

vivianliu0 commented:
@kouthar Ready for design review when you have a chance :)

ahmedhamodi commented on src/Components/Admin/Rooms/roomCard.js:
```diff
@@ -53,94 +122,119 @@ export default function RoomCard({ data, handleEditClick, handleDeleteClick }) {
-            justify="flex-start"
-            className={classes.card}
-        >
+        <ClickAwayListener onClickAway={handleClickAway}>
```
Genuinely never used the `ClickAwayListener` library before... nice find!

aaronabraham311 replied:
Yeah +1!

ahmedhamodi commented on src/Components/Admin/Rooms/roomCard.js:
```diff
@@ -7,43 +7,112 @@ import IconButton from "@material-ui/core/IconButton";
+        borderLeftWidth: "2px",
+        borderRightWidth: "2px",
+        borderTopWidth: "2px",
+        borderLeftColor: Colours.MainRed5,
+        borderRightColor: Colours.MainRed5,
+        borderTopColor: Colours.MainRed5,
```
Nice use of the new themes folder!

ahmedhamodi reviewed (commented):
From a code perspective, this looks great.

There are issues with different screen sizes, which Vivian and I talked about offline. Created a ticket to address this as it is due to logic that was pre-existing and not due to Vivian's code.

Will test locally and see if anything odd happens.

Pending design approval. :) Good work Vivian 🎉

aaronabraham311 commented on src/Components/Admin/Rooms/rooms.js:
```diff
@@ -26,10 +31,10 @@ export default function EscapeRooms({
     return (
         <Container component={"main"} maxWidth="lg">
             <CssBaseline />
+            <Typography component="div" variant="h5" className={classes.title}>
+                Your escape rooms
+            </Typography>
```
I believe this should be bolded (probably h4/h3) when I look at Figma

vivianliu0 replied:
Thanks, updated and bolded the title. h4/h3 increase in font sizes by quite a bit compared to h5, I think h5 is the closest fit
(image of h4)
![image](https://user-images.githubusercontent.com/56518575/126878133-a5ba06bb-59ed-41be-9806-af1a6660478c.png)

aaronabraham311 reviewed (changes requested):
Nice work as always Vivian! Couple of things:
- Can we add some top and bottom padding to the card options when you click on the three dot menu? I recommend you check out the CSS on Figma
- Can we remove the 'Admin Dashboard' text at the top?

kouthar commented:
![image](https://user-images.githubusercontent.com/32654489/126876745-212a2852-2f28-44f0-855f-4fa931eb93de.png)
![image](https://user-images.githubusercontent.com/32654489/126876760-fd225c38-3d37-4ec0-a2bc-59ae376f1124.png)
First photo: card near the bottom - bottom right aligned
Second photo: demonstrating alignment explicitly, but obvi don't show the kebab button when the menu is open lol

![image](https://user-images.githubusercontent.com/32654489/126876790-abf0905a-41ee-4306-a885-ecfca95eb638.png)
![image](https://user-images.githubusercontent.com/32654489/126876807-6d59afa9-fd07-4929-871d-e881c7a5d1ed.png)
Third and Fourth: card near the top/middle - top right aligned

vivianliu0 commented:
Upd

> Nice work as always Vivian! Couple of things:
> 
> * Can we add some top and bottom padding to the card options when you click on the three dot menu? I recommend you check out the CSS on Figma
> * Can we remove the 'Admin Dashboard' text at the top?

Updated, you can check out the second loom video I added to the PR to see the changes

vivianliu0 commented:
> ![image](https://user-images.githubusercontent.com/32654489/126876745-212a2852-2f28-44f0-855f-4fa931eb93de.png)
> ![image](https://user-images.githubusercontent.com/32654489/126876760-fd225c38-3d37-4ec0-a2bc-59ae376f1124.png)
> First photo: card near the bottom - bottom right aligned
> Second photo: demonstrating alignment explicitly, but obvi don't show the kebab button when the menu is open lol
> 
> ![image](https://user-images.githubusercontent.com/32654489/126876790-abf0905a-41ee-4306-a885-ecfca95eb638.png)
> ![image](https://user-images.githubusercontent.com/32654489/126876807-6d59afa9-fd07-4929-871d-e881c7a5d1ed.png)
> Third and Fourth: card near the top/middle - top right aligned

aaronabraham311 reviewed (approved):
LGTM!

ahmedhamodi reviewed (approved):
lgtm
